spring-cloud-dataflow-admin-local/src/main/java/org/springframework/cloud/dataflow/admin/spi/local/OutOfProcessModuleDeployer.java

```diff
@@ -192,8 +210,13 @@ private Instance(ModuleDeploymentId moduleDeploymentId, int instanceNumber, Proc
 			this.moduleDeploymentId = moduleDeploymentId;
 			this.instanceNumber = instanceNumber;
 			builder.directory(workDir.toFile());
-			builder.redirectOutput(this.stdout = Files.createTempFile(workDir, "stdout_" + instanceNumber + "_", ".log").toFile());
-			builder.redirectError(this.stderr = Files.createTempFile(workDir, "stderr_" + instanceNumber + "_", ".log").toFile());
+			String workDirPath = workDir.toFile().getAbsolutePath();
+			this.stdout =
+					Files.createFile(FileSystems.getDefault().getPath(workDirPath, "stdout_" + instanceNumber + ".log")).toFile();

```

Comment:
As for the API usage, it's not trivial to create a non-system specific Path relative to another Path, so a String absolute path has to be used. Anyway, simplified this a bit.